src/components/EditPost/EditPost.jsx

```diff
@@ -48,23 +51,66 @@ const EditPost = ({
   const handleGenerateButtonClick = () => {
     if (inputText) {
       getOpenAIResponse();
-      setActivePage("review");
+      // setActivePage("review");
     } else {
       alert("Please add input!");
     }
   };
 
+  const handlePostClick = () => {
+    setActivePage("review");
+  }
+
   return (
     <div className="promptpage__input-container">
-      <InputBox
-        value={inputText}
-        onChange={handleInputChange}
-        placeholder="Help me write a professional sounding post about my upcoming community event."
-        className="promptpage__input-box"
-      />
-      <Button className="generate" onClick={handleGenerateButtonClick}>
-        Generate
+      <PromptHeader headerText={"CommitAI Content Generator"} />
+      <div className="promptpage__sub-container">
+        <h4>Mood</h4>
+        <select>
+          <option>Tone1</option>
+          <option>Tone2</option>
+          <option>Tone3</option>
+        </select>
+        <h4>What do you want to share?</h4>
+        <InputBox
+          value={inputText}
+          onChange={handleInputChange}
+          placeholder="Help me write a professional sounding post about my upcoming community event."
+          className="promptpage__input-box"
+          labelVal="Give AI specific instructions for your content"
+        />
+        <h4>Output size</h4>
+        {/* this will need to be within form, along with everything else */}
+        <div className="promptpage__choice-selection">
+          <input type="radio" id="small" name="output-size" value="small"></input>
+          <label htmlFor="small">Small</label><br></br>
+          <input type="radio" id="med" name="output-size" value="med"></input>
+          <label htmlFor="med">Medium</label><br></br>
+          <input type="radio" id="large" name="output-size" value="large"></input>
+          <label htmlFor="large">Large</label><br></br>
+        </div>
+      </div>
+      <Button className="promptpage__generate-btn" onClick={handleGenerateButtonClick}>
+        Generate Social Post
       </Button>
+      <PromptHeader headerText={"Content"} />
+      <div className="promptpage__sub-container">
+        <ReviewPost
+          previewText={previewText}
+          setPreviewText={setPreviewText}
+        // setActivePage={setActiveTab}
+        />
+      </div>
+      <PromptHeader headerText={"Publishing Options"} />
+      <div className="promptpage__choice-selection">
+        <Button className="promptpage__generate-btn">
+          Save Draft
+        </Button>
+        <Button className="promptpage__post-btn" onClick={handlePostClick}>
+          Post
```

Comment:
This would be a design question, but I think this button should say Preview instead - what do you think? We could propose that to their team